### ISSUE TITLE
ensure proper heading order

### ISSUE THREAD

**Issue by Lane** [Hyperobjekt/untd-site#106]
headings should follow proper nesting / semantics (see https://www.w3.org/WAI/tutorials/page-structure/headings/)

**current heading order:**

![image](https://user-images.githubusercontent.com/21034/122294163-a140b000-ceac-11eb-8105-06acba9081d7.png)

**heading order should be something like:**

- h1: UNTX Social Mobility Hub (site name)
  - h2: The neighborhoods you live in have a major impact on social mobility
  - h2: What is socioeconomic mobility?
  - h2: 1. Research Library
    - h3: Curious to learn about what drives socioeconomic mobility?
  - h2:  Data Explorer
    - h3: Want to get a sense for the current conditions of communities across North Texas?
  - h2: Pathways to Mobility
    - h3: What are some strategies that could help move the needle?

**implementation notes: **

- use `<span>` tags inside of heading tags for styling purposes, instead of breaking a single heading into multiple heading tags

**example**

❌ splitting headings into pieces with separate heading tags
```html
<h1>The neighborhoods</h1>
<h2>you live in have a major impact on</h2>
<h1>social mobility</h1>
```

✅ split headings into pieces with span tags
```html
<h2>
  <span>The neighborhoods</span>
  <span>you live in have a major impact on</span>
  <span>social mobility</span>
</h2>
```

## Other Pages

These pages should have their headings structured as follows

### Research Library

- h1: The Research Library
  - h2: Research Library Topics
    - h3: Socioeconomic Mobility
    - h3: Family
    - h3: Quality Education
    - h3: Neighborhoods
    - h3: Health and Trauma
    ...

### Pathways to Mobility

- h1: Pathways to Mobility
  - h2: Interactive Worksheets
    - h3: Pathway One
      - h4: Students helping nonprofit redesign their programs to align with best practices related to increasing socioeconomic mobility 
    - h3: Pathway Two
      - h4: Students helping nonprofit redesign their programs to align with best practices related to increasing socioeconomic mobility

### FAQ

- h1: Frequently Asked Questions
  - h2: What is the NTX Social Mobility Hub, and why was it created?
  - h2: ...remaining FAQ questions

### About

- h1: About (not visible on page, so use `sr-only` helper class
  - h2: OPPORTUNITIES FOR UPWARD MOBILITY VARY DRASTICALLY ACROSS NORTH TEXAS.
  - h2: DISRUPTING THE CYCLE MAY BE POSSIBLE.
  - h2: WE WANT TO HELP
  - h2: PUTTING THE RESEARCH TO WORK
  - h2: ABOUT THE CENTER FOR SOCIOECONOMIC MOBILITY THROUGH LEARNING

### Contact

- h1: Contact Us
